src/mesh/amr_loadbalance.cpp

```diff
@@ -559,7 +613,9 @@ void Mesh::RedistributeAndRefineMeshBlocks(ParameterInput *pin, ApplicationInput
         } else {
           size = bsc2f;
         }
-        recvbuf[rb_idx] = BufArray1D<Real>("recvbuf" + std::to_string(rb_idx), size);
```

Comment:
This is style, but I'd use `int size = (oloc.level = nloc.level) ? bssame : bsc2f;` here